src/Microsoft.Health.Fhir.Core/Features/Validation/IKnowSupportedProfiles.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Fhir.Core.Features.Validation
+{
+    public interface IKnowSupportedProfiles
+    {
+        IEnumerable<string> GetSupportedProfiles(string resourceType, bool disablePull = false);
```

Comment:
>disablePull [](start = 75, length = 11)

nit: instead of `disablePull` do you think `disableCacheRefresh` might be a more descriptive name here?